engine-comparison/dispatcher.sh

```diff
@@ -5,61 +5,98 @@
 . $(dirname $0)/../common.sh
 . ${SCRIPT_DIR}/common-harness.sh
 
+
 build_engine() {
-  # [[ ! -e ~/$FENGINE_CONFIGS_DIR/$1 ]] && echo "cant do" && exit 1
-  # All of this code will probably be in a separate script
 
   FENGINE_CONFIG=$1
-
-  echo "Building $FENGINE_CONFIG"
+  [[ ! -e $FENGINE_CONFIG ]] && echo \
+    "Error: build_engine function called for FENGINE_CONFIG=$FENGINE_CONFIG,\
+    but this file can't be found" && return
```

Comment:
This is true...weighing pros and cons, exiting may make more sense because it saves the VM trouble/charges of running a whole experiment without the precise configs specified. I'll make this an exit